Contribute/dotnet/dotnet-contribute.md

```diff
@@ -94,6 +94,9 @@ docs
             Program.vb
 ```
 
+> [!NOTE]
+> The language folders under snippets are not needed in the language guide area, where only one language is assumed.
```

Comment:
This is related to https://github.com/dotnet/docs/issues/19791#issuecomment-668072235